doc/example/reference.cpp

```diff
@@ -441,6 +438,9 @@ BOOST_AUTO_TEST_CASE( demonstrates_configuring_mock_objects )
     MOCK_EXPECT( c.method ).once().with( 0 ).in( s ).returns( 42 );
     MOCK_EXPECT( c.method2 ).never().with( "ok", mock::any );
     MOCK_EXPECT( c.method2 ).at_least( 2 ).in( s ).throws( std::runtime_error( "error !" ) );
+    BOOST_CHECK(c.method(0) == 42);
```

Comment:
Should this be a `BOOST_CHECK_EQUAL`?

Reply:
Oh yes. I meant to use BOOST_TEST the universal version of this which does statement splitting (and shows both sides of the == on failure)